visual-challenge/src/main/java/applitools/hackathon/pages/ApplitoolsSite.java

```diff
@@ -0,0 +1,27 @@
+package applitools.hackathon.pages;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import applitools.hackathon.entities.User;
+import applitools.hackathon.sections.LoginPage;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.JSite;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+
+import static com.epam.jdi.light.common.CheckTypes.MATCH;
+
+@JSite("https://demo.applitools.com")
+public class ApplitoolsSite {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ApplitoolsSite' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/app/42_/jdi-light/pullRequest?prid=4513258)